frontend/catalyst/jax_tracer.py

```diff
@@ -432,16 +432,12 @@ def _bind_native_controlled_op(qrp, op, controlled_wires, controlled_values):
             qrp.insert(op.wires, qubits2[: len(qubits)])
             qrp.insert(controlled_wires, qubits2[len(qubits) :])
         elif isinstance(op, qml.GlobalPhase):
-            qubits = qrp.extract(op.wires)
             controlled_qubits = qrp.extract(controlled_wires)
             qubits2 = gphase_p.bind(
-                *[*qubits, *op.parameters, *controlled_qubits, *controlled_values],
-                qubits_len=len(qubits),
-                params_len=len(op.parameters),
+                *[*op.parameters, *controlled_qubits, *controlled_values],
                 ctrl_len=len(controlled_qubits),
             )
-            qrp.insert(op.wires, qubits2[: len(qubits)])
-            qrp.insert(controlled_wires, qubits2[len(qubits) :])
+            qrp.insert(controlled_wires, qubits2)
```

Comment:
Just double checking, but do we have tests with & without control qubits to make sure these lines work in both cases?

Reply:
Yes, see `test_global_phase.py`. We have vanilla, nested inside region and control.